tests/sanity/ignore-2.10.txt

```diff
@@ -74,6 +74,11 @@ plugins/modules/zos_operator_action_query.py import-2.6!skip # Python 2.6 is uns
 plugins/modules/zos_ping.py validate-modules:missing-gplv3-license # Licensed under Apache 2.0
 plugins/modules/zos_ping.py compile-2.6!skip # Python 2.6 is unsupported
 plugins/modules/zos_ping.py import-2.6!skip # Python 2.6 is unsupported
+plugins/modules/zos_script.py compile-2.6!skip # Python 2.6 is unsupported
+plugins/modules/zos_script.py import-2.6!skip # Python 2.6 is unsupported
+plugins/modules/zos_script.py validate-modules:missing-gplv3-license # Licensed under Apache 2.0
+plugins/modules/zos_script.py validate-modules:parameter-type-not-in-doc # Passing args from action plugin
+plugins/modules/zos_script.py validate-modules:undocumented-parameter # Passing args from action plugin
```

Comment:
I know the team will be working on eliminating such ignore entries, do you think there was a way to avoid passing only action plugin args to the module? Could we use module options that are part of the spec across the plugin and module?

Reply:
There's only like 2 action plugin args passed to the module, I'll try to remove them

Reply:
There's only like 2 action plugin args passed to the module, I'll try to remove them

Reply:
I think the issue is when we pass an a var from the plugin to the module and its only use to track state/value for the module but not actually part of the modules arg spec. Its not a blocker, we can address it in the other issue, I am not even sure what the best solution is, I can see the value in using a variable between the action plugin and module that may never be seen by the arge parser. Maybe its better to let the next set of folks working on these to solve it, maybe there is a way we can enter them into the arg spec yet not impact our ansible module doc.

Reply:
I spoke to @AndreMarcel99 about this, as he is currently working on the ignore issues for zos_copy, and we concluded that it would be worth it if I removed those private args now. If you review the code again you'll see that we don't have these entries anymore on the ignores.

Ansible doesn't let you pass anything that isn't part of the `arg_spec`, so it becomes tricky to deal with certain private args that we use in the collection. At least for zos_script, I think that the module's code became more readable after this change.